benefits/enrollment/views.py

```diff
@@ -38,25 +38,35 @@ def token(request):
     if not session.enrollment_token_valid(request):
         agency = session.agency(request)
         payment_processor = agency.payment_processor
-        client = Client(
-            base_url=payment_processor.api_base_url,
-            client_id=payment_processor.client_id,
-            client_secret=payment_processor.client_secret,
-            audience=payment_processor.audience,
-        )
-        client.oauth.ensure_active_token(client.token)
 
         try:
+            client = Client(
+                base_url=payment_processor.api_base_url,
+                client_id=payment_processor.client_id,
+                client_secret=payment_processor.client_secret,
+                audience=payment_processor.audience,
+            )
+            client.oauth.ensure_active_token(client.token)
             response = client.request_card_tokenization_access()
         except Exception as e:
+            logger.debug("Error occurred while requesting access token", exc_info=e)
             sentry_sdk.capture_exception(e)
 
-            if isinstance(e, HTTPError) and e.response.status_code >= 500:
-                analytics.failed_access_token_request(request, e.response.status_code)
-                data = {"redirect": reverse(ROUTE_SYSTEM_ERROR)}
-                return JsonResponse(data)
+            if isinstance(e, HTTPError):
+                status_code = e.response.status_code
+
+                if e.response.status_code >= 500:
```

Comment:
Small suggestion to use the `status_code` variable here instead

Reply:
Thanks for catching. Done in [fc291ed](https://github.com/cal-itp/benefits/pull/2117/commits/fc291ed866e52c07681a64bd81977554e0188f96)